### ISSUE TITLE
Timeline for Roadmap / Project discontinued?

### ISSUE THREAD

**Issue by nikolasburk** [kadirahq/lokka#41]
The [roadmap in the README](https://github.com/kadirahq/lokka#future-development) states a few upcoming features but the whole project hasn't been updated for several months! Is it still the plan to implement these features or is the whole project discontinued? If so, would you mind adding a deprecation note?

**Comment by MrLoh** [comment]
There should really be a note in the readme that this is discontinued and what to use instead.

**Comment by nikolasburk** [comment]
Agreed @MrLoh! It's a bit unfortunate that Lokka is still one of the most referenced simple GraphQL client when it's obviously not maintained any more.

**I can recommend using [graphql-request](https://github.com/graphcool/graphql-request) instead!**

**Comment by CaitlinWeb** [comment]
Bumping, I made the mistake of trying out Relay and it's way overcomplicated for what I need. Lokka seemed perfect... until I saw it had been years since it was updated.

Does anyone know if there are plans to get new maintainers? Or is there a well-maintained and -supported fork?
